develop-docs/sdk/telemetry/traces/index.mdx

```diff
@@ -185,6 +224,43 @@ Depending on the platform, other default data may be included. (For example, for
 
 A transaction's sampling decision should be passed to all of its children, including across service boundaries. This can be accomplished in the `startChild` method for same-service children and using the `senry-trace` header for children in a different service.
 
+### Propagated Random Value
+
+To increase the chance of capturing complete traces when users return a new sample rate from the `tracesSampler` in backend services, we propagate the generated random value used by the SDK for computing the sampling decision instead of sampling with a different random value in every service. Therefore, across a trace every SDK uses the same random value.
+
+Without a `tracesSampler` callback, an SDK fully inherits sampling decisions for propagated traces. In this case, the presence of `sample_rand` in the DSC has no effect on the sampling decision. However, this behavior is subject to change in the future.
```

Comment:
Maybe a stupid question, but what's the DSC? Dynamic Sampling Callback?

Reply:
https://develop.sentry.dev/sdk/telemetry/traces/dynamic-sampling-context/

Reply:
🤦 thanks